install

```diff
@@ -107,6 +119,12 @@ function parseCommandLineArguments() {
                   --default-apache-app  default apache app to redirect to (defaults to projectmanager)
                   --sendgrid-api-key)   sendgrid apikey that will be used to send emails when there is an error in a cron backup job
                   --alert-email)        email address to send alerts to when a backup job fails
+                  --licensedir          directory for license executable and x-platform directory to initialize license checker
+                  --licenseenvironment  assigned environment type of the server for license purposes
```

Comment:
there are a few valid values - can those be given in the comments

Reply:
do you mean the QA, RELEASE, DEV options?